src/Plutarch/Extra/IsData.hs

```diff
@@ -73,7 +75,7 @@ import Prelude
 newtype ProductIsData a = ProductIsData {unProductIsData :: a}
 
 -- | Variant of 'PConstantViaData' using the List repr from 'ProductIsData'
-newtype PConstantViaDataList (h :: Type) (p :: S -> Type) = PConstantViaDataList h
+newtype DerivePConstantViaDataList (h :: Type) (p :: S -> Type) = DerivePConstantViaDataList h
```

Comment:
Likewise here -- what is the point of this?

Reply:
Not sure. It's from Connor's PR.